src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+
+    public FigureType getRundomFigure() {
+        int index = new Random().nextInt(FigureType.values().length);
+        return FigureType.values()[index];
+    }
+
+    public Figure getDefaultFigure() {
+        Figure figure = new Circle(Color.WHITE, 10);
+        return figure;

```

Comment:
```suggestion
    public Figure getDefaultFigure() {
        return new Circle(Color.WHITE, 10);
```

and lets make Color.WHITE and number 10 as constant fields

Reply:
ok, good solution!